### ISSUE TITLE
weights to pb file to use in android

### ISSUE THREAD

**Issue by ayazozz** [AlexeyAB/darknet#750]
Hi, how can convert weights to protobuf file to use android. Can you help me please?

**Comment by alonzoMag** [comment]
Yes you can do this by using [this](https://github.com/thtrieu/darkflow#save-the-built-graph-to-a-protobuf-file-pb) repository guidelines, the only thing here is that yolo3 is not implemented yet. 
Any other way of doing it (with yolo3 for example) is welcome though.
